.github/workflows/pull-request.yaml

```diff
@@ -11,6 +11,7 @@ jobs:
         with:
           fetch-depth: 0
       - run: |
+          brew install tree
```

Comment:
??

Reply:
tree command not found, have to brew install tree